frontend/i18next-parser.config.js

```diff
@@ -55,7 +54,7 @@ module.exports = {
   lineEnding: 'auto',
   // Control the line ending. See options at https://github.com/ryanve/eol
 
-  locales: Object.keys(SUPPORTED_LOCALES),
+  locales: Object.keys(PARSER_SUPPORTED_LOCALES),
```

Comment:
If we only need the keys, no reason for this to be a map with display names.

Reply:
Good point, thanks!